pkg/config/validation/validation.go

```diff
@@ -130,6 +133,18 @@ type AnalysisAwareError struct {
 	Parameters []interface{}
 }
 
+type OverlappingForHTTPRoute struct {
```

Comment:
please add some comments on this structure

Reply:
it's done